src/core.c/Order.pm6

```diff
@@ -11,6 +11,22 @@ sub ORDER(int $i --> Order) is implementation-detail {
 }
 
 proto sub infix:<cmp>($, $, *% --> Order:D) is pure {*}
+multi sub infix:<cmp>(\a, Junction:D $b) {
+    Perl6::Metamodel::Configuration.throw_or_die(
```

Comment:
What is this magic?  :-)

Reply:
I originally tried just `X::Cannot::Junction.new(...).throw`, which built fine, but the test case instead died with `Could not find symbol '&Junction' in 'X::Cannot'`.

Reply:
That means just you need to stub the class before using it.

````raku
my class X::Cannot::Junction { ... }
````

Reply:
@lizmat thanks for the right suggestion! `throw_or_die` is not meant to be used outside of the metamodel.